app/poros/event.rb

```diff
@@ -15,18 +17,22 @@ def initialize(data)
     @activity = data[:attributes][:activity]
     @friend_name = data[:meta][:friend_name]
     @friend_role = data[:meta][:friend_role]
+    @gym_name = data[:meta][:gym_name]
+    @yelp_gym_id = data[:meta][:yelp_gym_id]
   end
 
   # TODO: Update method to handle different timezones
   # Returns the date in the format of 'Wednesday, Sep 22, 2021 9:41pm'
   def format_date
-    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S').strftime('%A, %b %d, %Y%l:%M%P')
+    DateTime.strptime(date_time,
+                      '%Y-%m-%dT%H:%M:%S').strftime('%A, %b %d, %Y%l:%M%P')
   end
 
   # TODO: Update method to handle different timezones
-  # Returns the date in the format of 'Wed 09/22/21 9:41pm'
+  # Returns the date in the format of 'Wed 09/22/21- 9:41pm'
   def format_date_short
-    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S').strftime('%a %m/%d/%y%l:%M%P')
+    DateTime.strptime(date_time,
+                      '%Y-%m-%dT%H:%M:%S').strftime('%a %m/%d/%y-%l:%M%P')
   end
```

Comment:
This might have been a RuboCop autocorrect.  From a code styling perspective, I'd prefer:
```
DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S')
        .strftime('%a %m/%d/%y-%l:%M%P')
```

Reply:
Good find, thanks so much for circling back!  This was bugging me big time - will update accordingly.